packages/ap-storage/alliance_platform/storage/async_uploads/models.py

```diff
@@ -308,7 +308,6 @@ def __init__(
         download_params: dict | None = None,
         **kwargs,
     ):
-        super().__init__(*args, max_length=max_length, **kwargs)
```

Comment:
you need to leave this in

Reply:
We should probably stop passing a max length to this field anyway. Not required for charfield any more.